device/arista/x86_64-arista_7170_64c/Arista-7170-64C/buffers_defaults_t0.j2

```diff
@@ -10,37 +14,70 @@
 {%- macro generate_buffer_pool_and_profiles() %}
     "BUFFER_POOL": {
         "ingress_lossless_pool": {
-            "size": "33329088",
+            "size": "{{ ingress_lossless_pool_size }}",
             "type": "ingress",
-            "mode": "dynamic",
-            "xoff": "7827456"
+            "mode": "dynamic"
         },
-        "egress_lossy_pool": {
-            "size": "26663272",
-            "type": "egress",
+        "ingress_lossy_pool": {
+            "size": "{{ ingress_lossy_pool_size }}",
+            "type": "ingress",
             "mode": "dynamic"
         },
         "egress_lossless_pool": {
-            "size": "42349632",
+            "size": "{{ egress_lossless_pool_size }}",
             "type": "egress",
-            "mode": "static"
+            "mode": "dynamic"
+        },
+        "egress_lossy_pool": {
+            "size": "{{ egress_lossy_pool_size }}",
+            "type": "egress",
+            "mode": "dynamic"
         }
     },
     "BUFFER_PROFILE": {
-        "ingress_lossy_profile": {
+        "ingress_lossless_profile": {
             "pool":"[BUFFER_POOL|ingress_lossless_pool]",
             "size":"0",
-            "static_th":"11075584"
+            "dynamic_th":"0"
```

Comment:
the dynamic threshold definating is alpha = 2^dynamic_th. If the dynamic_th is zero, then alpha is 1. it does not guarantee lossless behavior by SAI definition.